### ISSUE TITLE
document how to end up with a "bare domain"

### ISSUE THREAD

**Issue by bnewbold** [bluesky-social/pds#103]
via @ngerakines:

> I wish it were more widely announced that it's much easier to create a PDS on "pds.yourdomain.com", create a temporary handle to "tmpyou.pds.yourdomain.com" and then use the change handle method to change it to "yourdomain.com" using a TXT record. The whole PDS bare domain thing really seems to be tripping people up.

we should mention this pattern here in the PDS README, and maybe also need to update self-hosting docs on the atproto website

**Comment by ChiefGyk3D** [comment]
Yeah it was really tripping me up as well to be honest this is good to know as I'm planning to roll my own server

**Comment by madmalkav** [comment]
That pattern may be ok for low number of users, but for PDS with a high number of users it may be undesirable?

**Comment by DanMyers300** [comment]
Glad I found this thread because I've been struggling with this for hours. This fixed my issue!

**Comment by knotbin** [comment]
If you've have a PDS with an account at the "bare domain", can you simply switch the subdomain or do you have to make a completely new PDS there and migrate your account?
